### PR TITLE
Icon heading

### PR DESCRIPTION
**Pull request by martypowell**
Addresses #72 

- Adds Icon Heading Component
- Adds `light` section

### CODE REVIEW

**Comment by theilman7** [comment]
@martypowell Reviewing the pull request now and will add comments with any feedback I have.

File: src/styles/partials/components/_section.scss [hunk 1]
```diff
@@ -1,58 +1,64 @@
...
+
+        h2 {
+            @include alt-h2-heading($gold);
+        }
+
+        p {
+            color: $white;
+        }
+
+        .dg_button {
```
Comment (theilman7) [inline src/styles/partials/components/_section.scss:18]: Consider organizing css properties in alphabetical order for consistency in the source.

File: src/styles/partials/components/_icon-heading.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        }
+    }
+
+    i,
+    .dg_icon-heading__image-container {
+        margin-bottom: $default-margin;
+        position: relative;
+
+        &:after {
+            border-top: $border-width-large solid $gray-light;
```
Comment (theilman7) [inline src/styles/partials/components/_icon-heading.scss:71]: Consider organizing css properties in alphabetical order for consistency in the source.

Comment (martypowell) [reply src/styles/partials/components/_icon-heading.scss:71]: so there are cases where i think it makes sense to split them up, but you are right. let me give this another shot.

File: src/styles/partials/components/_section.scss (hunk 1 above)
Comment (martypowell) [reply src/styles/partials/components/_section.scss:18]: addressed

File: src/styles/partials/components/_icon-heading.scss (hunk 2 above)
Comment (martypowell) [reply src/styles/partials/components/_icon-heading.scss:71]: @theilman7 gave it a shot, let me know what you think.

File: src/styles/partials/components/_section.scss (hunk 1 above)
Comment (theilman7) [reply src/styles/partials/components/_section.scss:18]: Looks good.

**Comment by danfox01** [comment]
Sent to @jdomasky for testing. Will check back in when that's complete.

**Review by danfox01 — CHANGES_REQUESTED** [review]
@martypowell 

- The line color on dark background should be `pale blue lightest`; it's currently `gray-light`
- Let's pump up the icon font size to `30px`. I think it's `22px` right now.

FYI @jdomasky @sdurphy @hburnopp

**Comment by martypowell** [comment]
> @martypowell
> 
> * The line color on dark background should be `pale blue lightest`; it's currently `gray-light`
> * Let's pump up the icon font size to `30px`. I think it's `22px` right now.
> 
> FYI @jdomasky @sdurphy @hburnopp

Adjusted.

**Comment by jdomasky** [comment]
Taking another look.
@martypowell @danfox01 @sdurphy

**Comment by jdomasky** [comment]
These two issues have been resolved.
@martypowell @danfox01 @sdurphy

>     * The line color on dark background should be `pale blue lightest`; it's currently `gray-light`
> 
>     * Let's pump up the icon font size to `30px`. I think it's `22px` right now.

**Review by danfox01 — APPROVED** [review]
Looks good.

**Pull request merged**